assets/js/modules/analytics-4/hooks/useAudienceTilesReports.js

```diff
@@ -305,7 +311,7 @@ export default function useAudienceTilesReports( {
 				desc: true,
 			},
 		],
-		limit: 4, // Limit is set to 4 so that (not set) can be filtered out and 3 cities remain to display.
+		limit: 3,
```

Comment:
As above, please revert this change.

```suggestion
		limit: 4, // Limit is set to 4 so that (not set) can be filtered out and 3 cities remain to display.
```